chart.js.d.ts

```diff
@@ -0,0 +1,605 @@
+
+declare module "chart.js" {
+  export = Chart
+}
+
+declare namespace Chart {
+
+  function Line(context: CanvasRenderingContext2D, args: {data: LineChartData, options?: LineChartOptions}): LineChartInstance;
+
+  function Bar(context: CanvasRenderingContext2D, args: {data: BarChartData, options?: BarChartOptions}): BarChartInstance;
+
+  function Radar(context: CanvasRenderingContext2D, args: {data: RadarChartData, options?: RadarChartOptions}): RadarChartInstance;
+
+  function PolarArea(context: CanvasRenderingContext2D, args: {data: PolarAreaChartData, options?: PolarAreaChartOptions}): PolarAreaChartInstance;
+
+  function Bubble(context: CanvasRenderingContext2D, args: {data: CircularChartData, options?: CircularChartOptions}): CircularChartInstance;
+
+  function Doughnut(context: CanvasRenderingContext2D, args: {data: CircularChartData, options?: CircularChartOptions}): CircularChartInstance;
+
+  const defaults: {
+    global: ChartOptions
+    line: LineChartOptions
+    bar: BarChartOptions
+    radar: RadarChartOptions
+    polarArea: PolarAreaChartOptions
+    doughnut: CircularChartOptions
+    pie: CircularChartOptions
+  };
+
+  // Check required, what fields does the defaultConfig object contain?
+  const scaleService: {
+    registerScaleType<T extends Scale>(key: string, scale: T, defaultConfig: Object): void
+  };
+
+  interface ChartData {
+    labels: Array<string>
+    datasets?: Array<ChartDataSet>
+  }
+
+  interface ChartDataSet {
+    data: Array<number>
```

Comment:
This doesn't have to always be numbers. For scatter charts it can be an object with `x` and `y` properties.